src/containers/ToolbarContainer.jsx

```diff
@@ -0,0 +1,11 @@
+import { connect } from 'react-redux';
+
+import Toolbar from '../components/Toolbar';
+
+const mapDispatchToProps = dispatch => ({
```

Comment:
I see what you're trying to do, but do you need this to be that generic? Honestly, I doubt that toolbar is going to change THAT often. Consider just making a specialized toolbar tailored to your use case and just  declare all buttons and their actions in there